From cda171512fc8dc0aa385290e1f7644ff222e5f18 Mon Sep 17 00:00:00 2001
From: Bian Naimeng <biannm@cn.fujitsu.com>
Date: Fri, 3 Jul 2015 17:27:52 +0900
Subject: [PATCH] pacemaker: do not build help

Signed-off-by: Bian Naimeng <biannm@cn.fujitsu.com>

---
 configure.ac | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/configure.ac b/configure.ac
index 1429b01..e930e33 100644
--- a/configure.ac
+++ b/configure.ac
@@ -574,10 +574,7 @@ if test x"${MAKE}" = x""; then
    AC_MSG_ERROR(You need (g)make installed in order to build ${PACKAGE})
 fi
 
-AM_CONDITIONAL(BUILD_HELP, test x"${HELP2MAN}" != x"")
-if test x"${HELP2MAN}" != x""; then
-   PCMK_FEATURES="$PCMK_FEATURES generated-manpages"
-fi
+AM_CONDITIONAL(BUILD_HELP, test x"Do not build help" = x"")
 
 MANPAGE_XSLT=""
 if test x"${XSLTPROC}" != x""; then
